new-chatbot-app/prisma/schema.prisma

```diff
@@ -0,0 +1,55 @@
+// This is your Prisma schema file,
```

Comment:
I think we should remove these prisma for now. Because we haven't touched the database in this CancelReservationTool. Let's put it in for future PRs.